aiocache/backends/redis.py

```diff
@@ -8,11 +8,18 @@
 from aiocache.serializers import JsonSerializer
 
 
+is_aioredis_before_major_one = aioredis.__version__.startswith('0.')
```

Comment:
Can you update the `setup.py` file to reflect the new supported version? Now the range is set to `'aioredis>=0.3.3,<1'`

Reply:
Also, can you make the name less verbose? something like `aioredis_before_one`

Reply:
I think supporting version 1.0.0 beta 2 (`'aioredis>=0.3.3,<=1.0.0b2'`) by default isn't a good idea until is stable.

If someone wants to use aiocache with aioredis 1.0.0b2 should define explicitly:
```
aioredis==1.0.0b2
aiocache
```
instead of
```
aiocache[redis]
```

Reply:
Sounds good to me